pkg/sdn/plugin/egress_network_policy.go

```diff
@@ -13,6 +14,10 @@ import (
 )
 
 func (plugin *OsdnNode) SetupEgressNetworkPolicy() error {
+	if err := CheckDNSResolver(); err != nil {
+		return err
```

Comment:
Do we really want openshift to fail entirely if dig isn't present? (Rather than just making EgressNetworkPolicy not work?)

(Likewise in proxy.go)